### PR TITLE
26 report create kondratiuk

### PR DESCRIPTION
**Pull request by andrii-kondratiuk**

### CODE REVIEW

File: src/components/single-place/report-section/report-modal/report-modal.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      const { placeId } = this.props;
+
+      api
+        .report({
+          userJwt: jwt,
+          report: report,
+          placeId: placeId
+        })
+        .then(() => {
+          console.log('Success');
```
Comment (Dmitry528) [inline src/components/single-place/report-section/report-modal/report-modal.js]: Console)

File: src/components/single-place/report-section/report-modal/report-modal.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          this.setState({
+            report: '',
+            reportError: false,
+            selected: false
+          });
+        })
+        .catch(err => {
+          console.log(err);
```
Comment (Dmitry528) [inline src/components/single-place/report-section/report-modal/report-modal.js]: Console)

File: src/components/single-place/report-section/report-section.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import ReportModal from './report-modal/report-modal';
+import './report-section.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFlag } from '@fortawesome/free-regular-svg-icons';
+
+export default class ReportSection extends Component {
+  constructor(props) {
+    super(props)
+    this.hideModal = this.hideModal.bind(this)
```
Comment (Dmitry528) [inline src/components/single-place/report-section/report-section.js]: I think bind is bad idea) We have Es6)

File: src/components/single-place/report-section/report-modal/report-modal.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+  onSubmit = (e) => {
+    e.preventDefault();
+    const { report } = this.state;
+    if (report.length <= 2) {
+      this.setState({ reportError: true });
+    } else {
+      this.setState({ reportError: false });
+
+      const jwt = localStorage.getItem('token');
```
Comment (AndriiDruzhytskyi) [inline src/components/single-place/report-section/report-modal/report-modal.js:24]: Use  services/token-service for get or remove user tokens, please.

File: src/components/single-place/report-section/report-section.js (hunk 3 above)
Comment (marrybbb) [reply src/components/single-place/report-section/report-section.js]: Yes, you can just delete it )You use arrow fuunction)
https://stackoverflow.com/questions/52979915/why-we-dont-need-to-bind-the-arrow-function-in-react

File: src/components/single-place/report-section/report-modal/report-modal.js (hunk 1 above)
Comment (andrii-kondratiuk) [reply src/components/single-place/report-section/report-modal/report-modal.js]: Done

File: src/components/single-place/report-section/report-modal/report-modal.js (hunk 2 above)
Comment (andrii-kondratiuk) [reply src/components/single-place/report-section/report-modal/report-modal.js]: Done

File: src/components/single-place/report-section/report-section.js (hunk 3 above)
Comment (andrii-kondratiuk) [reply src/components/single-place/report-section/report-section.js]: Thank you for advise. Done.

**Review by Dmitry528 — APPROVED** [review]
Tested, good work

**Review by marrybbb — APPROVED** [review]

**Review by Viktor-Hordynskyi — APPROVED** [review]

**Pull request merged**